core/events.go

```diff
@@ -249,11 +249,13 @@ func (em *Events) handlePosEvent(e events.Event) {
 		em.resetOnMouseDown()
 	case events.MouseDrag:
 		if em.spriteSlide != nil {
+			e.SetHandled()
```

Comment:
This seems like the wrong approach to me. Instead of setting everything to handled, why not just prevent propagation of events that get sent to modeless dialogs directly in the central processing logic? Setting everything to handled seems like it will have a lot of bad consequences.